platform/darwin/src/MGLOfflineStorage.h

```diff
@@ -238,6 +238,31 @@ typedef void (^MGLOfflinePackRemovalCompletionHandler)(NSError * _Nullable error
  */
 @property (nonatomic, readonly) unsigned long long countOfBytesCompleted;
 
+/**
+ Sets a URL template for the given scheme.
```

Comment:
Gah, don’t know how I missed that. 😆 I do think it would be helpful, for those who aren’t familiar with the resources that make up a style, to know exactly what “any URLs” entails. But that points to the need for a conceptual “what’s a style” guide in the documentation.